ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -473,14 +480,16 @@ export default class ConfirmTransactionBase extends Component {
                 }
                 detailTitleColor={COLORS.BLACK}
                 detailText={
-                  <div className="confirm-page-container-content__currency-container">
-                    {renderHeartBeatIfNotInTest()}
-                    <UserPreferencedCurrencyDisplay
-                      type={SECONDARY}
-                      value={hexMinimumTransactionFee}
-                      hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
-                    />
-                  </div>
+                  !isMultiLayerFeeNetwork && (
```

Comment:
This does not change the code, only hides this component on `mutlilayerFeeNetwork`s. The multiline diff is just due to indentation changes.